wasted-app/RestaurantRegistrationControl.cs

```diff
@@ -38,15 +40,20 @@ private void registerButton_Click(object sender, EventArgs e)
         {
             var username = usernameInput.Text;
             var password = passwordInput.Text;
-            String error = validateUsernamePassword(username, password);
-            if (error == "")
+            String credentialError = getValidationError(username, password);
+
+            if (credentialError == "")
             {
-                passwordError.Text = "";
+                this.passwordError.Text = "";
+                ServicesController controller = ServicesController.Instance;
+                Credentials creds = new Credentials(username, password);
+                controller.RestaurantService.Register(creds, new Restaurant("aa", "Hello World", new Coords(10, 20), new Credentials()));
```

Comment:
should add proper id, not "aa".
should add a new field for the restaurant name, not "Hello World".

Reply:
When we'll change to databases the id field will be generated automatically